astroquery/jplhorizons/core.py

```diff
@@ -811,7 +1066,7 @@ def _parse_horizons(self, src):
                 try:
                     M1 = float(HGline[1].rstrip('M2'))
                     k1 = float(HGline[3].rstrip('k2'))
-                except ValueError:
+                except ValueError as e:
```

Comment:
no need to define `e` if it's not used for anything else. Otherwise using `err` or `exc` might be better than the single letter variable (though I admit we don't always have consistency with it).

Reply:
I removed `e`